contrib/editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
```

Comment:
Actually, just noticing this; should this be limited to `.go` files? We use spaces for Markdown (`.md`)